docs/docs/libraries/rust/api_reference.md

```diff
@@ -0,0 +1,10 @@
+# API Reference
+
+You can visit the [Rust documentation](https://wallet-lib.docs.iota.org/docs/doc/iota_wallet/index.html) to learn more about the API Reference.
```

Comment:
@Thoralf-M is wallet.rs also available on docs.rs - or do we need to deploy and link it here again?

Reply:
No, we need to deploy it